cmd/dcrtime/dcrtime.go

```diff
@@ -397,17 +638,42 @@ func _main() error {
 		return flagError
 	}
 
+	var mainnetHost string
+	var testnetHost string
+	var mainnetPort string
+	var testnetPort string
+	var upload func([]string, map[string]string) error
+	var download func([]string) error
+
+	// Check if API version flag is set to v1. If not, default
+	// to v2.
+	if hasAPIVersionFlag() && *apiVersion == v1.APIVersion {
```

Comment:
Do you check for an invalid API version?

I don't like defaulting to v2 if the api version is invalid.  This can lead to headaches on the user side.  If they pass up an invalid api version, you need to tell them that they passed up an invalid api version.

Reply:
Fixed mate, thanks for catching this :+1: